lib/honeybadger-api/fault.rb

```diff
@@ -2,8 +2,9 @@ module Honeybadger
   module Api
     class Fault
 
-      attr_reader :id, :project_id, :klass, :action, :component, :message,
-        :environment, :notices_count, :comments_count, :last_notice_at, :created_at
+      attr_reader :id, :project_id, :klass, :action, :component, :message, :assignee,
+        :environment, :notices_count, :comments_count, :last_notice_at, :created_at, :ignored,
+        :resolved
```

Comment:
@ajish Can you update the spec with the appropriate missing tests for `assignee`, `ignored` and `resolved` please?